src/framework/scene-grab.js

```diff
@@ -80,13 +80,20 @@ class SceneGrab {
         });
     }
 
-    allocateRenderTarget(renderTarget, device, format, isDepth, mipmaps, isDepthUniforms) {
+    resizeCondition(originalRenderTarget, requestedRenderTarget, device) {
+        if (originalRenderTarget && requestedRenderTarget) {
+            return originalRenderTarget.width !== requestedRenderTarget.width || originalRenderTarget.height !== requestedRenderTarget.height;
```

Comment:
why not just have code like above
```
const width = source ? source.colorBuffer.width : device.width,
const height = source ? source.colorBuffer.height : device.height,
```
and then compare it to requestedRenderTarget?

Reply:
it would make it more readable